### ISSUE TITLE
Agriculture Productivity After 2050

### ISSUE THREAD

**Issue by jiangyongye** [JGCRI/gcam-core#380]
Hi GCAM team : ),
I have a question about Agriculture Productivity Growth in GCAM
According to Doc, GCAM's Agriculture productivity growth is coming from FAO's 'Projected yields through 2050 for agricultural commodities'
![image](https://github.com/JGCRI/gcam-core/assets/49706086/8dac891c-b175-41f0-8fdf-59253eb27ad2)
Therefore, I just wonder how the Agriculture productivity growth after 2050 is calculated. Is it just using the same slope as the previous period(2015 - 2050)?

**Comment by jiangyongye** [comment]
Also, I noticed that different SSPs have different ag prod growth in GCAM, which is close to reality. But why are SSP1 ag prod growth exactly the same as SSP5 ag prod growth(different file but exactly the same code)?
![image](https://github.com/JGCRI/gcam-core/assets/49706086/b0d2c0dc-a214-4064-9218-3909b3593f4b)
Appreciate helps in advance!

**Comment by pkyle** [comment]
The post-2050 assumptions just follow a uniform, modest yield improvement rate, the same in all regions, set in `inst/extdata/aglu/A_defaultYieldRate.csv`. These rates are also used in region/crop combinations that aren't covered in the FAO projections (which tend to be minor crops anyways).
Re: the SSP1 vs. SSP5, we used a few different conventions for distinguishing SSP-specific XML input files; for example, an advanced technology assumptions XML file might be suffixed `_adv.xml` and called in the `scenarioComponents`of the configuration file for SSP1, and other times, such as here, in order to remove any ambiguity about which XML files map to which SSPs, we just suffixed each XML file by the SSP it was intended to correspond to, even if that means writing the same exact XML file twice, with different filenames. In any case, the SSPs are created by the combination of specific XML files and the configuration files, which are identified in `exe/batch_SSP_REF.xml`.
